frame/airdrop/src/models.rs

```diff
@@ -0,0 +1,65 @@
+use codec::{Decode, Encode, MaxEncodedLen};
+use composable_support::types::{
+	CosmosEcdsaSignature, CosmosPublicKey, EcdsaSignature, EthereumAddress,
+};
+use scale_info::TypeInfo;
+use sp_runtime::{MultiSignature, RuntimeDebug};
+
+/// A single Airdrop.
+#[derive(Encode, Decode, PartialEq, Copy, Clone, TypeInfo, MaxEncodedLen)]
+pub struct Airdrop<AccountId, Balance, Moment> {
+	/// Creator of the Airdrop.
+	pub(crate) creator: AccountId,
+	/// Total funds committed to the Airdrop.
+	pub(crate) total_funds: Balance,
+	/// Total number of recipients
+	pub(crate) total_recipients: u32,
+	/// Amount of the `total_funds` already claimed.
+	pub(crate) claimed_funds: Balance,
+	/// Starting block of the Airdrop.
+	pub(crate) start: Option<Moment>,
+	/// The minimum time, in blocks, between recipient claims.
+	pub(crate) schedule: Moment,
+	/// Set `true` if an airdrop has been explicitly disabled.
+	pub(crate) disabled: bool,
+}
```

Comment:
curious - why are the fields pub(crate), but the struct is pub?

Reply:
26e0882f9dc37e7e9ca229985415a3cded986b9a